app/presenters/field_of_operation_presenter.rb

```diff
@@ -6,7 +6,7 @@ class FieldOfOperationPresenter < ContentItemPresenter
   def title_and_context
```

Comment:
I think the change to this file might want to be fixed up with the earlier commit where all the other presenter changes were.